codes/classical/bits/repetition code.yml

```diff
@@ -0,0 +1,58 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+code_id: repetition
+physical: bits
+logical: bits
+
+name: 'Binary repetition code'
+
+introduced: '\cite{doi:10.1201/9781315147901}'
+
+description: 'This code encodes 1-bit information into a \(n\)-bit string. \(n\) needs to be an odd number, since receiver will pick the majority at decoding stage. Its basic idea is to increase the code distance by repeating the logical information several times. It is a \((n,1)\)-Hamming code.'
```

Comment:
Remove "Hamming". You mean to say that repetition codes live in the Hamming space, which is correct, but in this case one does not say "a Hamming code".